twas-nd/src/main/scripts/install.sh

```diff
@@ -54,6 +54,22 @@ mkdir im_installer
 unzip -q "$IM_INSTALL_KIT" -d im_installer
 ./im_installer/userinstc -log log_file -acceptLicense -installationDirectory ${IM_INSTALL_DIRECTORY}
 
+# Check whether IBMid is entitled or not
```

Comment:
We should add [bash strict mode](https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425) to all our scripts.